src/controllers/accounts/AccountsAssetsController.ts

```diff
@@ -0,0 +1,108 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+import { BadRequest } from 'http-errors';
+
+import { validateAddress } from '../../middleware';
+import { AccountsAssetsService } from '../../services/accounts';
+import AbstractController from '../AbstractController';
+
+/**
+ * Get asset information for an address.
```

Comment:
These are nice but in the future you don't have to do them. They are legacy from before we had the openapi docs.

Reply:
Good to know :). Yea i noticed them on the other controllers, and thought it wouldn't hurt.